samples/linux/jobs_sample/aws_iot_config.h

```diff
@@ -71,7 +71,7 @@
 #define DISABLE_METRICS false ///< Disable the collection of metrics by setting this to true
 
 // TLS configs
-#define IOT_SSL_READ_TIMEOUT 10 ///< Timeout associated with underlying socket of TLS connection (set by mbedtls_ssl_conf_read_timeout)
+#define IOT_SSL_READ_TIMEOUT 3 ///< Timeout associated with underlying socket of TLS connection (set by mbedtls_ssl_conf_read_timeout)
```

Comment:
Is this in ms as well, If so can you please add that to the name to indicate the unit as well?